README.md

```diff
@@ -65,104 +60,38 @@ py.test -s tests/trading_tests.py
-set to the contract’s address. Finally, the addresses are added to the 
-whitelist, and contracts are compiled. If all contracts compile successfully, 
-any temporary files are deleted. Otherwise, contract writers must re-upload
-the necessary files after fixing errors.
-
-## API Maker Scripts
-
-The API Maker script parses through files ending with the token “data_api”. 
-It extracts contract API locally from a file, and remotely from GitHub. It 
-reads existing contract API info from augur-contracts/api.json, and then 
-adds the newly obtained information to the the existing information. Subsequently,
-all contracts’ API information is updated. Finally, all contract API is 
-put together in a file and printed to the screen.
+The security model is based on the idea that reporters will want to invest in a market where honesty prevails, and will not trust fraudulent markets. For example, assume that there exist two almost identical universes: an honest universe, where the factually correct answer is the predominant answer, and a fraudulent universe, where incorrect answers are predominant in order to cheat the system. The REP in the fraudulent universe will approach 0, and the REP in the honest universe will approach the theoretical yield. This phenomenon occurs because reporters do not want to trade on markets in a dishonest universe. Therefore, they will not create markets in a dishonest universe. Ultimately, the REP holders will not receive any market fees.
+
+In order to create a fraudulent universe, an attacker must fork the main branch. They will need to buy a large share of REP and stake it on the lie outcome, as nobody else will want to engage in a deceitful market. In principle, the attacker will lose up to 2.5 times the maximum profit made from a successful attack, and their REP will approach 0.
+
+The reentrancy guard (libraries/ReentrancyGuard.sol) and controller (Controller.sol) contracts also help uphold security. One can also attack Ethereum contracts by creating a chain of wanted contract calls through a malicious user. The mutex contract prevents against that by preventing someone from re-entering the contracts before they return. The controller contract serves to update Augur and stores a registry of whitelisted contract addresses. The controller also can perform administrative actions, such as transferring ownership and performing emergency stop functions. It can be operated in either the developer mode, where the developer updates contracts, or the decentralized mode, when people must upload contracts to make changes and therefore, the market creators/traders must cancel orders and shift to the new contract. For bugs found involving money or REP, contracts are locked / an escape hatch is enabled which allows withdrawal of shares [and cancelling orders] for a value half-way between the bid-ask spread at the time of the lock. In addition, another escape hatch is enabled, which allows withdrawal of any locked up REP.
```

Comment:
Small nit, this sentence sounds a bit repetitive: 
```
The mutex contract prevents against that by preventing someone from re-entering the contracts before they return.```
Recommend changing it to something like: 
```The mutex contract protects against that by preventing someone from re-entering the contracts before they return.```

Reply:
Fixed.